example_scripts/galaxy_zoo_touch_table/prepare_db_from_classification_export.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import pandas as pd
+import json
+import string
+from tqdm import tqdm
+from itertools import cycle
+
+def preprocessing():
+    # Read the local CSV subject export
+    classifications=pd.read_csv("classification-export.csv")
+
+    classifications['metadata']=classifications['metadata'].map(lambda x: json.loads(x))
+    classifications['locations']=classifications['locations'].map(lambda x: json.loads(x))
```

Comment:
Excellent. Assigning back like this saves space in large dataframes; I don't know how big this one gets but it's a good practice to follow, provided you don't need the original column. Bravo.